### ISSUE TITLE
Indication of what is happening

### ISSUE THREAD

**Issue by clapierre** [daisy/ace-gui#23]
When you are working on a title it would be great to get some indication of % done? (if possible) otherwise at least what it is currently working on as you had with the command line version
Just a scroll window with the output from the command line would probably work here so you see its still working and not just hung on a large file.

**Comment by rdeltour** [comment]
Right. A percentage may be too difficult as it's hard to get a good estimate of the total duration.
For now `ace-core` doesn't provide an API to get detailed progress feedback, so we need to implement that there first in order for the GUI to get the info. I love the idea however!
